tests/fixtures/snapshots/stupicat-table-with-escapes-output

```diff
@@ -0,0 +1,3 @@
+|First|Second|Third|Fourth|
```

Comment:
I think this document should also contain examples where `|` should not be escaped, for completeness.
After all, I would hope that other tests would already fail if the `in_table_cell` condition would not be present.

Reply:
I don’t think there’s any existing coverage of the `` `|` `` case—I didn’t see any, at least!—but I’ll add some coverage for cases of `` `|` `` in scenarios which are *not* part of the table.

Reply:
That's what I meant, thank you.

Reply:
Perfect. Just pushed up that and all the other changes!